funcbench/bench.go

```diff
@@ -67,7 +68,7 @@ func (b *Benchmarker) benchOutFileName(commit plumbing.Hash) (string, error) {
 	return fmt.Sprintf("%s-%s.out", bb.String(), commit.String()), nil
 }
 
-func (b *Benchmarker) execBenchmark(pkgRoot string, commit plumbing.Hash) (out string, err error) {
+func (b *Benchmarker) execBenchmark(ctx context.Context, pkgRoot string, commit plumbing.Hash) (out string, err error) {
```

Comment:
```suggestion
func (b *Benchmarker) exec(ctx context.Context, pkgRoot string, commit plumbing.Hash) (out string, err error) {
```
We are in benchmark so it's kind of clear. Yes. I know I did in the first place (: But good ideas come with time.